nautobot/extras/views.py

```diff
@@ -2028,8 +2018,17 @@ def get_extra_context(self, request, instance):
         }
 
 
-class ScheduledJobDeleteView(generic.ObjectDeleteView):
-    queryset = ScheduledJob.objects.all()
+class ScheduledJobApprovalQueueListView(ObjectListViewMixin):
+    queryset = ScheduledJob.objects.needs_approved()
+    table_class = tables.ScheduledJobApprovalQueueTable
+    filterset_class = filters.ScheduledJobFilterSet
+    filterset_form_class = forms.ScheduledJobFilterForm
+    action_buttons = ()
+
+    def get_template_name(self):
+        if self.action == "list":
+            return "extras/scheduled_jobs_approval_queue_list.html"
+        return super().get_template_name()
```

Comment:
When (if ever) will the action *not* be `list`?

Reply:
If that is the case I think it would make sense to auto raise an error, something like 

```python
if self.action != "list":
    raise Something("some descriptive message")
```